apm-agent-plugins/apm-grails-plugin/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-agent-plugins</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.16.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apm-grails-plugin</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+
+    <properties>
+        <apm-agent-parent.base.dir>${project.basedir}/../..</apm-agent-parent.base.dir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.grails</groupId>
+            <artifactId>grails-web-mvc</artifactId>
+            <version>[4.0.1,)</version>
```

Comment:
in supported versions in doc, you've written it was working for versions 3+, and here we test only with 4.0+, thus I guess you've tested with both 3.x and 4.0.x. What happens with 2.x ? Is it supported or is it just ignored by this instrumentation (for example it does not have the same classes/interfaces instrumented by this plugin) ?

Reply:
> thus I guess you've tested with both 3.x and 4.0.x

Yes, I've tested that manually. The reason I didn't set version `[3.0.0,)` is that maven would download every single minor and bugfix version between 3.0.0 and now which would take forever.

> What happens with 2.x ?

Grails 2.x is excluded via the `classLoaderCanLoadClass("org.grails.web.mapping.mvc.GrailsControllerUrlMappingInfo")` matcher